uint16[] indices # table of output indices which states are defined
bool[] states
---